docs/docs/external/getting-started/test-network/overview.md

```diff
@@ -0,0 +1,32 @@
+---
+id: overview
+title: Overview of Test Network Components
+---
+
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+
+Weaver offers a basic test network launching capability, both to demonstrate interoperation modes and to serve as a testbed for development and prototyping. Different modes (or scenarios) require different sets of components, but collectively you will need to run the following:
+
+- [Fabric testnet](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/tests/network-setups/fabric/dev) - A pair of basic Fabric networks for testing interop flows
+- [Corda testnet](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/tests/network-setups/corda) - A basic Corda network for testing interop flows
+- [Relay](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/core/relay) - The daemon and protocol for cross-DLT interoperability. Two instances of this project is needed, one for the Corda network and one for the Fabric network
```

Comment:
I prefer changing daemon -> server (as that will be consistent with what have captured here https://github.com/hyperledger-labs/weaver-dlt-interoperability/blob/main/core/relay/relay-docker.md that the relay driver is run as a server). Also, daemon names typically end with a `d` and we don't want to call it `relayd`.

Reply:
We used to call it `relayd`, so this term has lingered. Fixed.